test/test_dynamic_shapes.py

```diff
@@ -12,12 +12,13 @@ def test_simple_expand(self):
     size2 = 2
```

Comment:
can we add a check in the beginning of this test to make sure `XLA_EXPERIMENTAL` is set?

Reply:
done